library/src/main/java/com/pokegoapi/util/Signature.java

```diff
@@ -79,9 +79,8 @@ public static void setSignature(PokemonGo api, RequestEnvelopeOuterClass.Request
 	}
 
 
-	private static int getLocationHash1(PokemonGo api, byte[] authTicket,
-										RequestEnvelopeOuterClass.RequestEnvelope.Builder builder) {
-		XXHashFactory factory = XXHashFactory.fastestInstance();
+	private static int getLocationHash1(PokemonGo api, byte[] authTicket, RequestEnvelopeOuterClass.RequestEnvelope.Builder builder) {
+		XXHashFactory factory = XXHashFactory.safeInstance();
```

Comment:
yeah, i merged another PR on my branch! im gonna clean it up once i finish it and also strip out the un-needed throws! can you just check if it works meanwhile? im waiting for some score